### PR TITLE
feat(ffe-account-selector-react): Added types for Account-selector components

### PR DESCRIPTION
**Pull request by JonasJore**
Added types for Account-selector components

### CODE REVIEW

File: packages/ffe-account-selector-react/src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { Component, HTMLAttributes, ChangeEventHandler } from 'react';
+import { Account } from './util/types';
+
+interface AccountSelectorProps extends HTMLAttributes<HTMLDivElement> {
```
Comment (hdaljord) [inline packages/ffe-account-selector-react/src/index.d.ts]: This should not inherit from HTMLAttributes<HTMLDivElement> as extra props defined in HTMLAttributes<HTMLDivElement> are not passed into any div element created by AccountSelector.

File: packages/ffe-account-selector-react/src/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    onChange: ChangeEventHandler<HTMLInputElement>;
+    selectedAccount: Account;
+    showBalance?: boolean;
+    value: string;
+    readonly: boolean;
+}
+
+declare class AccountSelector extends Component<AccountSelectorProps, any> {}
+
+interface AccountSelectorMultiProps extends HTMLAttributes<HTMLDivElement> {
```
Comment (hdaljord) [inline packages/ffe-account-selector-react/src/index.d.ts]: See comment above about AccountSelector

File: packages/ffe-account-selector-react/src/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+interface AccountSelectorProps extends HTMLAttributes<HTMLDivElement> {
+    accounts?: Array<Account>;
+    className?: string;
+    id: string;
+    locale: Array<String>;
+    noMatches?: string;
+    onAccountSelected: ChangeEventHandler<HTMLInputElement>;
+    onChange: ChangeEventHandler<HTMLInputElement>;
+    selectedAccount: Account;
```
Comment (hdaljord) [inline packages/ffe-account-selector-react/src/index.d.ts]: selectedAccount is optional in propTypes

File: packages/ffe-account-selector-react/src/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    showBalance?: boolean;
+    value: string;
+    readonly: boolean;
```
Comment (hdaljord) [inline packages/ffe-account-selector-react/src/index.d.ts]: readonly is optional in proptypes

File: packages/ffe-account-selector-react/src/index.d.ts [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    suggestionsFilter: Function;
+    onSelect: Function;
+    value: string;
+    locale: Array<String>;
+    shouldHideSuggestionsOnSelect: boolean;
+    shouldSelectHighlightedOnTab: boolean;
+    shouldHideSuggestionsOnBlur: boolean;
+    shouldHideSuggestionsOnReset: boolean;
+    shouldShowSuggestionsOnFocus?: boolean;
+    onSuggestionSelect?: Function;
```
Comment (hdaljord) [inline packages/ffe-account-selector-react/src/index.d.ts]: onSuggestionSelect is required in propTypes

File: packages/ffe-account-selector-react/src/index.d.ts [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    onChange?: Function;
```
Comment (hdaljord) [inline packages/ffe-account-selector-react/src/index.d.ts]: More accurate type should be ChangeEventHandler<HTMLInputElement>

File: packages/ffe-account-selector-react/src/index.d.ts [hunk 7]
```diff
@@ -0,0 +1,71 @@
+import { Component, HTMLAttributes, ChangeEventHandler } from 'react';
+import { Account } from './util/types';
```
Comment (hdaljord) [inline packages/ffe-account-selector-react/src/index.d.ts]: Will typescript understand the Account-type defined in this file?

**Review by hdaljord — CHANGES_REQUESTED** [review]

File: packages/ffe-account-selector-react/src/index.d.ts (hunk 7 above)
Comment (JonasJore) [reply packages/ffe-account-selector-react/src/index.d.ts]: No, it will not. Made an `Account` type for this in the `index.d.ts` -file

**Review by hdaljord — APPROVED** [review]
Looks good to me :)

**Pull request merged**